python/pyspark/sql/connect/dataframe.py

```diff
@@ -1068,6 +1068,23 @@ def inputFiles(self) -> List[str]:
         query = self._plan.to_proto(self._session.client)
         return self._session.client._analyze(query).input_files
 
+    def toDF(self, *cols: str) -> "DataFrame":
```

Comment:
I found hard to support `def toDF(self, *cols: "ColumnOrName") -> "DataFrame":` for the Connect case.

How can we convert a Column to a string? This PySpark API is quite different from Scala side as Scala API only supports a list of strings ` def toDF(colNames: String*): DataFrame`.

Reply:
I suspect the typing `ColumnOrName ` in PySpark is incorrect

Reply:
^ yes

Reply:
cc @HyukjinKwon

Reply:
Yeah, should fix them.

Reply:
https://github.com/apache/spark/pull/38834